graphql/tests/query/projectEvents/results.json

```diff
@@ -20,7 +20,7 @@
                   "githubWebhooksEnabled": false,
                   "vars": {
                     "vars": {
-                      "hello": "world"
+                      "hello": "{REDACTED_BEFORE}"
```

Comment:
Can we add an case asserting that the after is an empty string for vars that have not changed?